codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/restxml/AWSHttpProtocolRestXMLCustomizations.kt

```diff
@@ -6,22 +6,11 @@
 package software.amazon.smithy.aws.swift.codegen.restxml
 
 import software.amazon.smithy.aws.swift.codegen.AWSHttpProtocolCustomizations
-import software.amazon.smithy.aws.swift.codegen.AWSHttpRequestXMLEncoder
-import software.amazon.smithy.aws.swift.codegen.AWSHttpResponseXMLDecoder
 import software.amazon.smithy.swift.codegen.integration.ClientProperty
 
 class AWSHttpProtocolRestXMLCustomizations : AWSHttpProtocolCustomizations() {
 
     override fun getClientProperties(): List<ClientProperty> {
-        val properties = mutableListOf<ClientProperty>()
-        val requestEncoderOptions = mutableMapOf<String, String>()
-        val responseDecoderOptions = mutableMapOf<String, String>()
-        responseDecoderOptions["dateDecodingStrategy"] = ".secondsSince1970"
-        responseDecoderOptions["nonConformingFloatDecodingStrategy"] = ".convertFromString(positiveInfinity: \"Infinity\", negativeInfinity: \"-Infinity\", nan: \"NaN\")"
-        responseDecoderOptions["trimValueWhitespaces"] = "false"
-        responseDecoderOptions["removeWhitespaceElements"] = "true"
-        properties.add(AWSHttpRequestXMLEncoder(requestEncoderOptions))
-        properties.add(AWSHttpResponseXMLDecoder(responseDecoderOptions))
-        return properties
+        return listOf<ClientProperty>()
```

Comment:
These properties are all related to encoder/decoder config, so none are needed for RestXML.